src/metabase/models/action.clj

```diff
@@ -149,7 +143,7 @@
                      (select-keys implicit-action [:kind]))))
            actions))))
 
-(defn select-actions
+(defn select-actions-without-implicit-params
```

Comment:
looks like we could make this private? same for `implicit-action-parameters`.

Reply:
👍